includes/class-edd-license-handler.php

```diff
@@ -210,6 +227,54 @@ public function license_help_text( $active_tab = '' ) {
 
 	}
 
+	/**
+	 * If the supplied license key is for a pass, updates the `edd_pass_licenses` option with
+	 * the pass ID and the date it was checked.
+	 *
+	 * Note: It's intentional that the `edd_pass_licenses` option is always updated, even if
+	 * the provided license data is not for a pass. This is so we have a clearer idea
+	 * of when the checks started coming through. If the option doesn't exist in the DB
+	 * at all, then we haven't checked any licenses.
+	 *
+	 * @since 2.10.6
+	 *
+	 * @param string $license
+	 * @param object $api_data
+	 */
+	private function maybe_set_pass_flag( $license, $api_data ) {
+		$passes = get_option( 'edd_pass_licenses' );
+		$passes = ! empty( $passes ) ? json_decode( $passes, true ) : array();
+
+		if ( ! empty( $api_data->pass_id ) && ! empty( $api_data->license ) && 'valid' === $api_data->license ) {
```

Comment:
The `$api_data` is the entire remote response array, including headers, so this is not ever true, from what I can tell here.

Reply:
Should be fixed now. https://github.com/easydigitaldownloads/easy-digital-downloads/pull/8679/commits/4c443f3ae9d4dfce31c5d9c38bda7560415979de